frontend/schemaCMS/src/modules/dataSource/dataSource.sagas.js

```diff
@@ -293,6 +293,38 @@ function* updateOne({ payload: { requestData, dataSource } }) {
       return;
     }
 
+    if (
+      requestData[DATA_SOURCE_API_URL] ||
+      requestData[DATA_SOURCE_API_JSON_PATH] ||
+      requestData[DATA_SOURCE_API_JSON_PATH] === '' ||
+      requestData[DATA_SOURCE_AUTO_REFRESH] ||
+      requestData[DATA_SOURCE_AUTO_REFRESH] === false
```

Comment:
If you checking here if variable is only defined boolean then you can use ramda and `is(Boolean,  requestData[DATA_SOURCE_AUTO_REFRESH])`, same above `is(String, requestData[DATA_SOURCE_API_JSON_PATH])`